SETUP.md

```diff
@@ -0,0 +1,36 @@
+```
+mvn clean package dependency:copy-dependencies  -DincludeArtifactIds=fluo-recipes-core
```

Comment:
Is this file still needed?  It looks like it was abandoned

Reply:
Oh yeah, it was created and abandoned while I was working on the branch.